interfaces/cython/cantera/examples/multiphase/adiabatic.py

```diff
@@ -51,15 +53,15 @@
 
     tad[i] = mix.T
     print('At phi = {0:12.4g}, Tad = {1:12.4g}'.format(phi[i], tad[i]))
-    xeq[:,i] = mix.species_moles
+    xeq[:, i] = mix.species_moles
 
 # write output CSV file for importing into Excel
 csv_file = 'adiabatic.csv'
-with open(csv_file, 'w') as outfile:
+with open(csv_file, 'w', newline='') as outfile:
```

Comment:
This is recommended in the [Python docs](https://docs.python.org/3/library/csv.html#id3). I believe the reason is because the CSV writer will handle newlines itself and this prevents newlines in the values from being interpreted incorrectly.

Reply:
I suppose... On the other hand, every example in the Python docs uses this form, so I'm inclined to include it here. Do you have a strong opinion that it should be removed?

Reply:
No, it can stay. I'm just being cranky about having to figure out the details of the Python `csv` module in order to review a PR that is supposedly about YAML files.

Reply:
Yeah, that's fair. In the future, feel free to tell me to split the PR into several